velox/docs/functions/spark/bitwise.rst

```diff
@@ -12,6 +12,17 @@ Bitwise Functions
     Returns the bitwise OR of ``x`` and ``y`` in 2's complement representation.
     Corresponds to Spark's operator ``^``.
 
+.. spark:function:: bit_count(x) -> integer
+
+    Returns the number of bits that are set in the argument ``x`` as an unsigned 64-bit integer,
+    or NULL if the argument is NULL.
+
+.. spark:function:: bit_get(x, pos) -> byte
+
+    Returns the value of the bit (0 or 1) at the specified position.
+    The positions are numbered from right to left, starting at zero.
+    The valid position argument should in range [0, bits of x), otherwise throw exception.
```

Comment:
> The valid position argument should in range [0, bits of x), otherwise throw exception.

Perhaps, "The value of 'pos' argument must be between 0 and number of bits in 'x' - 1. Invalid 'pos' values result in an error."